# Problema 1

## 1.1

In [6]:
M = 5.9e27      # g
R = 6.4e8       # cm
G = 6.67e-8     # dyn*cm^2/g^2
d = 200         # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 0.0006 cm/s**2


## 1.2

In [7]:
d = 1.28e9 # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 3843.0664 cm/s**2


## 1.3.a


In [9]:
M = 1.9e33 # g
d = 200 # cm
R = 1e7  # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 50692000.0000 cm/s**2


## 1.3.b

In [10]:
print(a/979)

51779.36670071501


Sí, esta es igual a 51779 veces la aceleración de la gravedad, y un ser humano sería destrozado y espaghettificado